src/fillTank.test.js

```diff
@@ -1,11 +1,130 @@
+/* eslint-disable max-len */
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('Should be called', () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it('Should tank to max capacity, if customer want to much ', () => {
```

Comment:
The test description 'Should tank to max capacity, if customer want to much' contains a typo. It should be 'wants too much'.